### PR TITLE
Fix point orientation operation in mapper node. Update simulation documentation

### PR DESCRIPTION
**Pull request by Alescontrela**
Fixed some stuff in the mapper node. Created an optional `plot_rviz` flag in mapper.launch that sets up an rviz visualization when `plot_rviz:=true`.

### CODE REVIEW

File: igvc_navigation/launch/localization.launch [hunk 1]
```diff
@@ -21,7 +21,7 @@
 
       <!-- for if the IMU (after magnetic declination adjustment)
            does not read 0 -->
-      <param name="yaw_offset" value="1.57"/>
+      <param name="yaw_offset" value="0"/>
```
Comment (JasonGibson274) [inline igvc_navigation/launch/localization.launch]: We should double check this on the robot. This may be a on the robot configuration. In that case we should rotate our IMU in the URDF to match whatever is on the robot. 

TLDR; check IMU orientation IRL and make sure simulation matches not the other way around.

File: igvc_navigation/launch/pather.launch [hunk 2]
```diff
@@ -3,7 +3,7 @@
 <!-- map.launch -->
 <launch>
     <!-- Waypoints -->
-    <include file="$(find igvc_navigation)/launch/waypoint_source.launch" />
+    <!-- <include file="$(find igvc_navigation)/launch/waypoint_source.launch" /> -->
```
Comment (JasonGibson274) [inline igvc_navigation/launch/pather.launch]: so this is a simulation thing that breaks IRL. If we want to restructure things so that this is not an issue now is the time for that.

File: igvc_rviz_plugins/configs/jessii_conf.rviz [hunk 3]
```diff
@@ -0,0 +1,289 @@
+Panels:
```
Comment (JasonGibson274) [inline igvc_rviz_plugins/configs/jessii_conf.rviz]: We should have more of these but I am not sure how to best organize them. Opening up an issue for it

File: igvc_navigation/launch/localization.launch (hunk 1 above)
Comment (Alescontrela) [reply igvc_navigation/launch/localization.launch]: So I talked to justin about this yesterday. He mentioned we no longer need the 'yaw_offset' parameter. The reason it was needed at competition was because the IMU wasn't given enough time to converge on an accurate yaw value, so it set its initial yaw to an unfiltered, inaccurate value - this required manual correction via the yaw_offset parameter.

File: igvc_navigation/launch/pather.launch (hunk 2 above)
Comment (Alescontrela) [reply igvc_navigation/launch/pather.launch]: I agree. A simple workaround would be to include a launch file flag, `simulation`, that doesn't load IRL waypoints when set to true.

Let me know if you think this would be a good approach.

File: igvc_rviz_plugins/configs/jessii_conf.rviz (hunk 3 above)
Comment (Alescontrela) [reply igvc_rviz_plugins/configs/jessii_conf.rviz]: So the reason i used this specific configuration is because it highlights everything the robot is capable of doing at the moment:
* sensing with the lidar
* creating an occupancy grid
* planning and following paths using the occupancy grid

If you want, i can remove it from this PR but I think it adds value as a simple debugging tool.

File: igvc_navigation/launch/localization.launch (hunk 1 above)
Comment (JasonGibson274) [reply igvc_navigation/launch/localization.launch]: Then I would remove the parameter entirely, but it was mostly used for actual offset.

File: igvc_rviz_plugins/configs/jessii_conf.rviz (hunk 3 above)
Comment (JasonGibson274) [reply igvc_rviz_plugins/configs/jessii_conf.rviz]: No leave it in. I like it. I just mean later on we should organize them

File: igvc_navigation/launch/pather.launch (hunk 2 above)
Comment (JasonGibson274) [reply igvc_navigation/launch/pather.launch]: yes, let's go with that for now

File: igvc_navigation/launch/localization.launch (hunk 1 above)
Comment (Alescontrela) [reply igvc_navigation/launch/localization.launch]: o.k. i'll get rid of it. I'll still include it as a parameter in the documentation just for reference.

**Comment by Alescontrela** [comment]
The new PR should include the above-mentioned changes 👍
